eth_account/types.py

```diff
@@ -0,0 +1,31 @@
+from typing import (
+    Dict,
+    Sequence,
+    TypedDict,
+    Union,
+)
+
+from eth_keys.datatypes import (
+    PrivateKey,
+)
+from eth_typing import (
+    HexStr,
+)
+from hexbytes import (
+    HexBytes,
+)
+
+Blobs = Sequence[Union[bytes, HexBytes]]
+Bytes32 = bytes
+PrivateKeyType = Union[Bytes32, int, HexStr, PrivateKey]
+
+
+# from web3.types
+class AccessListEntry(TypedDict):
+    address: HexStr
+    storageKeys: Sequence[HexStr]
+
+
+AccessList = Sequence[AccessListEntry]
+
+TransactionDictType = Dict[str, Union[AccessList, bytes, HexStr, int]]
```

Comment:
Agreed...I wonder if it's worth moving the *web3.py* `TxParams` to *eth_typing* and using it here as well since we're getting ready for a beta round before `v7` there?

Reply:
Don't mind me, I think these are different expected types.